### DISCUSSION TITLE
Pimcore data importer stuck at 0/x items processed...

### DISCUSSION THREAD

**Discussion by AmiralSaucisse** [pimcore/pimcore#11865] [Q&A / Support] [answered]
Hi, i'm importing a small amount of data in .xlsl in a small test class composed of 4 fields, again just for testing before importing the +13000 row file.
i've followed the youtube setup tuto (https://www.youtube.com/watch?v=5iCtrzL3LUA),
apart from the element loading parameter (i'm in Id with the unique identifier of my product as the Id.)
i read the doc but as you can imagine, i'm a newbie on an internship so i'm a little deasperate.

After saving, i can lunch the import but it get stuck at 0/x items processed even after letting it run all night long

do someone have an idea of why i cant import any data ?

if my config has no chance of working in this condition, what can i do to import this xlsx in the good data object ?

Edit : i'm using pimcore w/ Docker

**Comment by kingjia90** [comment, marked as answer]
have you set the [cronjob](https://github.com/pimcore/data-importer/blob/1.x/doc/01_Installation.md#command-based)? 
manually try `bin/console datahub:data-importer:process-queue-parallel --processes=5` or `datahub:data-importer:process-queue-sequential ` (depending on your config) in the docker php container to see if it get processed.

**Reply by AmiralSaucisse** [reply]
Hello, effectively at this time I haden't setup the cron, i tried a normal composer install pimcore + data importer without docker and then found out about the cron, then i've return on docker and setup the cron, but the import still doesn't work, 

finnaly i've returned to my non-docker install, setup the cron there, but the import work only if i chmod 777 / chauwn www-data:www-data a certain file in var to let the import write(and publish), i don't know how to resolve this problem but it work fo now 
i dont know anything about security but i do know that chmod 777 isn't the right thing to do.

Thanks for your answer !

**Reply by kingjia90** [reply]
I am glad that you figured it out and solved it on your own.
For the chmod permission, it depends on how you set it up and i am unable to help you there, but i believe it would work with "lower combinations" than 777, as long as execute permissions are set.

I am closing this QA at it feels resolved.
